zebra/zebra_rib.c

```diff
@@ -3082,8 +3081,11 @@ void rib_update_table(struct route_table *table, rib_update_event_t event)
 				 * gateway, NHT will
 				 * take care.
 				 */
-				if (nh)
+				if (nh) {
+					SET_FLAG(re->status,
+							ROUTE_ENTRY_CHANGED);
```

Comment:
indentation looks a little off here?

Reply:
That's how clang-format modified it so I left it alone.

Reply:
fixing the white space and re-pushing